tracer/test/Datadog.Trace.Tests/Agent/ApiTests.cs

```diff
@@ -38,7 +38,7 @@ public async Task SendTraceAsync_200OK_AllGood()
             factoryMock.Setup(x => x.GetEndpoint(It.Is<string>(s => s == TracesPath))).Returns(new Uri("http://localhost/traces"));
             factoryMock.Setup(x => x.GetEndpoint(It.Is<string>(s => s == StatsPath))).Returns(new Uri("http://localhost/stats"));
 
-            var api = new Api(apiRequestFactory: factoryMock.Object, statsd: null, updateSampleRates: null, partialFlushEnabled: false);
```

Comment:
This is the only one you set `propagateProcessTags: true` for - is that intentional?